stories/Icon/Icon.story.jsx

```diff
@@ -84,6 +93,21 @@ storiesOf('Foundation', module).add('Icons', () => (
           </small>
 
           <CodeExample code={exampleCode} />
+
+          <Title as="h2">Unavailable Icons</Title>
+
+          <p>
+            If if the string passed in the `Icon` is unavailable in the
+            catalogue it is going to display only the string wrapped in a `span`
+            element and no icon will render.
+          </p>
+          <small>
+            {exampleIncorrectIcons.map(icon => (
+              <IconWrapper key={icon.props.name}>{icon}</IconWrapper>
```

Comment:
We have equal keys (info) in map.
![image](https://user-images.githubusercontent.com/3389749/76628167-163f7000-651b-11ea-8544-3b3f21fb92b4.png)

Ps: Te conflict is because we have two mappings and this key is equal of above